### ISSUE TITLE
Threshold for ElasticSearch FreeStorageSpaceTooLowAlarm is too low

### ISSUE THREAD

**Issue by philof** [awslabs/aws-solutions-constructs#153]
The comment and description for the alarm for ElasticSearch's FreeStorageSpace indicates the alarm is for 20 GiB of free space.

```
// FreeStorageSpace minimum is <= 20480 for 1 minute, 1 consecutive time
alarmDescription: 'A node in your cluster is down to 20 GiB of free storage space.'
```

https://github.com/awslabs/aws-solutions-constructs/blob/5636629bf77a996115e7a244d584e4864c08a49c/source/patterns/%40aws-solutions-konstruk/core/lib/elasticsearch-helper.ts

However, the alarm threshold is set to 2000, or 2 GiB of free storage space.

### Environment
  - **AWS Solutions Constructs Version     : v1.83.0**

--- 

This is :bug: Bug Report

**Comment by biffgaut** [comment]
Thanks - we'll take a look.

**Comment by biffgaut** [comment]
This will be fixed in our next release. Thanks.

**Comment by biffgaut** [comment]
Released in 1.94.0
